NDNc/pipeline/pipeline-interests-aimd.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+* N-DISE: NDN for Data Intensive Science Experiments
+* Author: Catalin Iordache <catalin.iordache@cern.ch>
+*
```

Comment:
Please run clang-format tool on all modified files like this: 

**clang-format --style=file -i pipeline/pipeline-interests.hpp** 

from NDNc directory, where the *.clang-format* file is